Users/Install.rst

```diff
@@ -83,10 +83,77 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is the most user friendly method of managing virtual environments and
```

Comment:
IMO words such as `most` sud be avoided.

Reply:
Considered. Let me fix it.